content/src/ui/Component.NetworkError.json

```diff
@@ -0,0 +1,13 @@
+{
+  "en": {
+    "title": "Network Errors",
+    "message": "App fails to establish or maintain a connection with the server."
```

Comment:
How about "Lost internet connection" / " It seems like you don't have any connection to the internet at the moment"?